downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,6 +60,21 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
+- Introduced the `Extract to transform function` code action to extract a value expression into a transform function.
```

Comment:
What is meant by "value expression"?

Reply:
Just checked the example in the PR. May make sense to elaborate on where this CA is available rather than trying to use an arbitrary term to cover this.

Reply:
Used the term to describe the value expression in a field of a mapping constructor, as specified in https://ballerina.io/spec/lang/master/#section_6.7.2:~:text=%5D%20variable%2Dname-,value%2Dexpr,-%3A%3D%20expression%0Acomputed. Added more details to clarify this.